Assets/Scripts/AnnotationTool.cs

```diff
@@ -70,17 +66,51 @@ public void Annotate(Vector3 nextPoint)
 
                 // Broadcast adding an annotation
                 SimulationEvents.Instance.AnnotationAdded.Invoke(
-                    currentAnnotation.transform.localPosition,
-                    currentAnnotation.transform.localRotation,
-                    currentAnnotation.transform.localScale,
+                    startPointForDrawing,
+                    endPointForDrawing,
                     currentAnnotation.name);
 
                 startPointForDrawing = Vector3.zero;
                 endPointForDrawing = Vector3.zero;
                 currentAnnotation = null;
+
             }
         }
     }
+
+    private void AddAnnotationLineRenderer(GameObject _currentAnnotation, Vector3 startPos, Vector3 endPos, Color playerColor)
+    {
+        _currentAnnotation.GetComponent<AnnotationLine>().StartPos = startPos;
+        _currentAnnotation.GetComponent<AnnotationLine>().EndPos = endPos;
+        _currentAnnotation.GetComponent<AnnotationLine>().RemoveStartPoint();
+
+        int pointCount = 30;
+
+        _currentAnnotation.layer = LayerMask.NameToLayer("Marker");
+
+        LineRenderer lineRendererArc = _currentAnnotation.AddComponent<LineRenderer>();
+        MeshCollider meshColliderArc = _currentAnnotation.AddComponent<MeshCollider>();
+        Mesh mesh = new Mesh();
+        lineRendererArc.useWorldSpace = false;
+        lineRendererArc.startWidth = annotationWidth * annotationWidthMultiplier;
+        lineRendererArc.endWidth = annotationWidth * annotationWidthMultiplier;
+        lineRendererArc.material = annotationMaterial;
+
+        lineRendererArc.positionCount = pointCount;
+        lineRendererArc.material.color = playerColor;
+        Vector3[] points = new Vector3[pointCount];
+        for (int i = 0; i < pointCount; i++)
+        {
+            points[i] = Vector3.Slerp(startPos, endPos, (float)i / (pointCount - 1));
```

Comment:
I had to look this up:
https://docs.unity3d.com/ScriptReference/Vector3.Slerp.html
`Spherically interpolates between two vectors.`

How do we know the direction of the sphere interpolation (inward vs outward)?

Reply:
Good question.  It appears to use the smaller, less than 180 degree path.  But that is something to keep an eye on as I don't see a way to specify it.